quesma/stats/ingest_statistics_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
```

Comment:
This "new" file is just renamed `quesma/stats/index_statistics_test.go` with a couple `IndexStatistics` changed to `IngestStatistics`.